Gifski/Base.lproj/MainMenu.xib

```diff
@@ -70,7 +70,7 @@
                             <menuItem title="Quick Look" keyEquivalent=" " id="Wfh-bM-KvX">
                                 <modifierMask key="keyEquivalentModifierMask"/>
                                 <connections>
-                                    <action selector="quickLook:" target="-1" id="Zr4-ny-Pim"/>
+                                    <action selector="quickLook:" target="-1" id="IQy-U1-llq"/>
```

Comment:
Just curious, what changed here?

Reply:
Unpinned & pinned the action again and so the id regenerated 🤕